File: leetcode/easy/1480.Running Sum of 1d Array.js
// https://leetcode.com/problems/running-sum-of-1d-array/description/

/**
 * @param {number[]} nums
 * @return {number[]}
 */

nums = [1, 2, 3, 4];
var runningSum = function (nums) {
    sum = 0;
    res = []
    for (let i = 0; i < nums.length; i++) {
        sum = sum + nums[i];
        res[i] = sum;
    }

    console.log(res);
};

runningSum(nums);

File: leetcode/easy/1480.Running Sum of 1d Array.py
# https://leetcode.com/problems/running-sum-of-1d-array/description/

from typing import List

nums = [5,8,2,7]

def runningSum(nums: List[int]) -> List[int]:
    sum = 0
    res = []
    for i in range(0, len(nums)):
        sum = sum + nums[i]
        res.append(sum)

    return res;

print(runningSum(nums))
